### PR TITLE
Remove css files and override the MUI components css using themes

### PR DESCRIPTION
**Pull request by Rohit0301**

### CODE REVIEW

File: src/components/ui/TextInput.tsx [hunk 1]
```diff
@@ -32,8 +32,8 @@ const TextInput: FC<Props> = ({
   otherProps = {},
 }) => {
   return (
-    <Box className="text-field-container">
-      <label htmlFor={id}>{label}</label>
+    <Box>
+      <InputLabel htmlFor={id}>{label}</InputLabel>
```
Comment (satishwaghela) [inline src/components/ui/TextInput.tsx:36]: Text field already has label, if you want different style then you can override.

Comment (Rohit0301) [reply src/components/ui/TextInput.tsx:36]: Screenshot 2024-10-30 at 2 53 51 PM

I want the UI like the left one, if we use the default label on MUI then the Ui looks like the right one. And MUI also use the seperate label tag for the left textfield

File: src/components/ui/CustomButton.tsx [hunk 2]
```diff
@@ -22,11 +21,27 @@ const CustomButton: FC<Props> = ({
   return (
```
[the remark this answers is not in the document]
Comment (Rohit0301) [reply src/components/ui/CustomButton.tsx:21]: I have created single component for both normal button and icon button

[the remark this answers is not in the document]
Comment (satishwaghela) [reply src/components/ui/CustomButton.tsx:21]: Ok but what's need of that?

File: src/components/ui/TextInput.tsx (hunk 1 above)
Comment (satishwaghela) [reply src/components/ui/TextInput.tsx:36]: Can't it be done using theme overrides?

**Pull request merged**